platform/node/src/node_expression.cpp

```diff
@@ -79,9 +79,7 @@ void NodeExpression::Parse(const Nan::FunctionCallbackInfo<v8::Value>& info) {
 
     auto success = [&cons, &info](std::unique_ptr<Expression> result) {
         auto nodeExpr = new NodeExpression(std::move(result));
-        const int argc = 0;
-        v8::Local<v8::Value> argv[0] = {};
-        auto wrapped = Nan::NewInstance(cons, argc, argv).ToLocalChecked();
+        auto wrapped = Nan::NewInstance(cons).ToLocalChecked();
```

Comment:
I am not familiar with Nan. Is this the same?

Reply:
Yes, it is. According to [this](https://github.com/nodejs/nan/blob/main/doc/maybe_types.md#api_nan_new_instance), Nan::NewInstance have 3 overloads. So calling with 0-length arguments is the same than omitting argc and argv parameters.